docker/trino/catalog/hive.properties

```diff
@@ -0,0 +1,15 @@
+connector.name=hive
+
+hive.allow-drop-table=true
+hive.ignore-absent-partitions=true
+hive.metastore.thrift.delete-files-on-drop=true
+hive.metastore.uri=thrift://hive-metastore:9083
+hive.metastore.username=admin
+hive.non-managed-table-writes-enabled=true
+hive.s3.aws-access-key=accesskey
+hive.s3.aws-secret-key=secretkey
+hive.s3.endpoint=http://hive-metastore-minio:9000
+hive.s3.path-style-access=true
+hive.s3select-pushdown.enabled=true
+hive.storage-format=PARQUET
+hive.timestamp-precision=MICROSECONDS
```

Comment:
It turns out that you can only have a single timestamp precision for all columns in hive. I'm not sure what's up with that.